openwisp_users/api/serializers.py

```diff
@@ -392,7 +392,7 @@ def validate_confirm_password(self, value):
             'confirm_password'
         ):
             raise serializers.ValidationError(
-                _('New password and Confirm password do not match.')
+                _('The two password fields didn’t match.')
```

Comment:
`didn’t` seems to contain a slant quote here, I suggest replacing it with a normal quote for consistency

Reply:
This slant quote is also present in Django https://github.com/django/django/blob/4816dc942860caf076c7c85ea9dbfa8bfab212ff/django/contrib/auth/forms.py#L397-L399